<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tracking</class>
 <widget class="QWidget" name="tracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tracker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:qlineargradient(spread:pad, x1:0.091, y1:0.101636, x2:0.991379, y2:0.977, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QPushButton" name="startbutton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 245, 244);
background-color: rgb(143, 240, 164);
font: 75 11pt &quot;Ubuntu Condensed&quot;;
border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 255), stop:0.373979 rgba(255, 255, 255, 255), stop:0.373991 rgba(33, 30, 255, 255), stop:0.624018 rgba(33, 30, 255, 255), stop:0.624043 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>Start Tracker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stoptracker_button">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 51, 59);
background-color: rgb(246, 97, 81);</string>
   </property>
   <property name="text">
    <string>Stop Tracker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh_button">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 245, 244);
font: 75 11pt &quot;Ubuntu Condensed&quot;;
border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 255), stop:0.373979 rgba(255, 255, 255, 255), stop:0.373991 rgba(33, 30, 255, 255), stop:0.624018 rgba(33, 30, 255, 255), stop:0.624043 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>â†» refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_ext_but">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>310</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 51, 59);
background-color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>Logout and exit</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>841</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="documentTitle">
    <string notr="true"/>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date - </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Time - </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_date">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Peddana</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>                                             W E L C O M E   TO   K E Y B O A R D   T R A C K E R</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_time">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
